pkg/lang/ir/v1/r.go

```diff
@@ -51,7 +51,6 @@ func (g generalGraph) installRPackages(root llb.State) llb.State {
 
 	// TODO(terrytangyuan): Support cache.
 	cmd := sb.String()
-	root = llb.User("envd")(root)
```

Comment:
I am wondering why we remove the statement.

Reply:
> I am wondering why we remove the statement.

The default path for installing R package is `/usr/local/lib/R/site-library` which is only writable to root. The old implementation of `install.r_pacakges()` will try to install to this path while `envd` user doesn't have permission.

Reply:
I just created another pull request from a different local branch. I will update the changes there and dispute this one

Reply:
Fixed, this pull request will be only about the e2e test for R environment @gaocegege